zerver/lib/export.py

```diff
@@ -142,6 +142,7 @@
     "zerver_realmemoji",
     "zerver_realmfilter",
     "zerver_realmplayground",
+    "zerver_realmuserdefault",
```

Comment:
I am not sure we would want to export this table, but just included it for now.

Reply:
Yes, we definitely do.  We should also be sure to import it in the `import_realm.py` logic.